Translation to PT-BR

DrZanuff opened the pull request:
Hey this game was just published here in Brazil. So I added translations to PT-BR. Great game and great tool.

DrZanuff committed:
inital commit - pt-br translation

DrZanuff committed:
translated Flood and Wheather

DrZanuff committed:
expansion translation

DrZanuff committed:
Create README.md

jpraet commented on index.html:
```diff
@@ -193,6 +193,7 @@
               <a id="lang-de" class="dropdown-item" href="#" onclick="selectLanguage('es')">Español</a>
               <a id="lang-fr" class="dropdown-item" href="#" onclick="selectLanguage('fr')">Français</a>
               <a id="lang-pl" class="dropdown-item" href="#" onclick="selectLanguage('pl')">Polski</a>
+              <a id="lang-pt-br" class="dropdown-item" href="#" onclick="selectLanguage('pt')">Português</a>
```
```suggestion
              <a id="lang-pt" class="dropdown-item" href="#" onclick="selectLanguage('pt')">Português</a>

```

jpraet committed:
id="lang-pt"

it should match the language parameter, the id is used to make it the active value in the dropdown.

jpraet committed:
Remove empty line accidentally added by github suggestion

jpraet committed:
Delete README.md for merging the PR

jpraet commented:
Thanks!
